docs/projects/ad9083_evb/index.rst

```diff
@@ -0,0 +1,316 @@
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI distributes the bit/elf files of these projects as part of the
+:dokuwiki:`ADI Kuiper Linux <resources/tools-software/linux-software/kuiper-linux>`.
+If you want to build the sources, ADI makes them available on the
+:git-hdl:`HDL repository </>`. To get the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository.
+
+Then go to the hdl/projects/$eval_board/$carrier location and run the make
+command.
+
+**Linux/Cygwin/WSL**
+
+Building the project on ZCU102 without parameters, will use the default
+configuration, meaning L = 4, M = 16 and L = 1.
+
+.. shell::
+
+   $cd hdl/projects/ad9083_evb/zcu102
+   $make
+
+Example of running the ``make`` command with parameters:
+
+.. shell::
+
+   $cd hdl/projects/ad9083_evb/zcu102
+   $make RX_JESD_L=4 RX_JESD_M=16
+
+The result of the build, if parameters were used, will be in a folder named
+by the configuration used:
+
+if the following command was run
+
+``make RX_JESD_L=4 RX_JESD_M=16``
+
+then the folder name will be:
+
+``RXL4_RXM16``
+because of truncation of some keywords so the name will not exceed the limits
+of the Operating System (``JESD``, ``LANE``, etc. are removed) of 260
+characters.
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user guide.
+
+Software considerations
+-------------------------------------------------------------------------------
+
+Resources
+-------------------------------------------------------------------------------
+
+Systems related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Another quick start guide using this evaluation board (but not using our HDL
+  code) is 
+  :dokuwiki:`AD9083-EVB evaluation using the ADS80V3EBZ capture board <resources/eval/ad9083>`
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Product datasheet: :adi:`AD9083`
+- `AD9083-EVB schematic <https://wiki.analog.com/_media/resources/eval/ad9083/02_059760b.pdf>`__
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :git-hdl:`AD9083-EVB HDL project source code <projects/ad9083_evb>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - AXI_DMAC
+     - :git-hdl:`library/axi_dmac`
+     - :ref:`axi_dmac`
+   * - AXI_SYSID
+     - :git-hdl:`library/axi_sysid`
+     - :ref:`axi_sysid`
+   * - SYSID_ROM
+     - :git-hdl:`library/sysid_rom`
+     - :ref:`axi_sysid`
+   * - UTIL_CPACK2
+     - :git-hdl:`library/util_pack/util_cpack2`
+     - :ref:`util_cpack2`
+   * - UTIL_ADCFIFO (used on A10SoC project)
+     - :git-hdl:`library/util_adcfifo`
+     - ---
+   * - UTIL_ADXCVR for AMD
```

Comment:
Instead of "for AMD" and "for Intel" wouldn't it be easier to mark them with */ ** and put a Legend below (keep the IP name without any comments)? Example: https://github.com/analogdevicesinc/hdl/blob/0747ae67c7a8ee4426d1028bc5bf7a30a46d3afa/docs/projects/ad7616_sdz/index.rst?plain=1#L330

Reply:
done